recipes/naiad/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "naiad" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.ifremer.fr/naiad/naiad/-/archive/{{ version }}/naiad-{{ version }}.tar.gz
+  sha256: f9cbdc51890c7831b2371b0ae37c7325342e24b92bb3d4b9826168e934802772
+  
+build:
+  script_env:
+   - POETRY_DYNAMIC_VERSIONING_BYPASS={{ version }}
+  script: 
+   - {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
```

This should fix Windows.

```suggestion
  noarch: python
  entry_points:
    naiad-create-index = naiad.cli.create:main
    naiad-inspect-csv = naiad.cli.csv:main
    naiad-search = naiad.cli.search:main
    naiad-crossover-search = naiad.cli.crosssearch:main
    naiad-register = naiad.cli.storing:main
    naiad-inquire = naiad.cli.inquire:main
```